### PR TITLE
ci: use our crdb image

### PR DESCRIPTION
**Pull request by hedwigz**

### CODE REVIEW

**Review by masseelch — CHANGES_REQUESTED** [review]
What about the `docker-compose.yml`?

**Comment by hedwigz** [comment]
> What about the `docker-compose.yml`?

`docker-compose.yml` has the `command` option so that I can start the cockroach image with `start-single-node --insecure` but the GitHub action service doesn't have this option so I built another image to solve that.

**Comment by a8m** [comment]
Can’t you use run.entrypoint or run.args for this? 
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsentrypoint

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -480,7 +480,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       cockroach:
-        image: timveil/cockroachdb-single-node:latest
+        image: hedwigz/cockroachdb-single-node:v21.2.11
```
Comment (a8m) [inline .github/workflows/ci.yml:483]: I think we can build it in the action and use it here, no?

**Review by a8m — APPROVED** [review]

**Pull request merged**